.gitignore

```diff
@@ -143,12 +143,10 @@ mhs-reference-implementation/selenium_tests/data/local_asid
 /mhs/spineroutelookup/nhs-spineroutelookup-env.yaml
 export-env-vars-and-run-mhs-docker.sh
 /integration-tests/integration_tests/integration-test-env.yaml
+/integration-tests/integration_tests/integration-tests-env.yaml
```

Comment:
Are there both files integration-test-env.yaml and integration-testa-env.yaml?

Reply:
removed the duplicate entry